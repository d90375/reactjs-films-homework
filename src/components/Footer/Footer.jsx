import React from "react";
import "./Footer.scss";

const Footer = () => {
  return (
    <div className="footer">
      <div className="footer__title">
        <span>FILMS</span>
      </div>
      <div className="footer__copyright">
        <span>Copyright Â© 2018 d90375</span>
      </div>
    </div>
  );
};

export default Footer;
