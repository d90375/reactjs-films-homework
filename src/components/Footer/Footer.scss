.footer {
  display: flex;
  flex-direction: column;
  align-items: center;
  width: 100%;
  height: 263px;
  box-shadow: 0 10px 40px rgba(0, 0, 0, 0.04);
  background-color: #fff;
}

.footer__title {
  margin-top: 75px;
  color: #0aaee4;
  font-family: "Helvetica Neue", serif;
  font-size: 50px;
  font-weight: 400;
  text-transform: uppercase;
  letter-spacing: 8px;
}

.footer__copyright {
  margin-top: 75px;
  opacity: 0.4;
  color: #242b2d;
  font-family: "Helvetica Neue", serif;
  font-size: 24px;
  font-weight: 400;
  letter-spacing: 1.2px;
}
