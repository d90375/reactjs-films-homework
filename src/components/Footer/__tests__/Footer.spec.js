import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Footer from "../Footer";

const renderer = new ShallowRenderer();
const setUp = (props) => {
  return renderer.render(<Footer {...props} />);
};

describe("Footer component", () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });

  it("should render Posts component", () => {
    expect(component).toMatchSnapshot();
  });
});
