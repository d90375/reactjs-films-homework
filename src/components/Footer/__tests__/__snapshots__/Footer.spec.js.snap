// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render Posts component 1`] = `
<footer
  className="footer"
>
  <div
    className="title"
  >
    <span>
      FILMS
    </span>
  </div>
  <div
    className="copyright"
  >
    <span>
      Copyright Â© 2018 d90375
    </span>
  </div>
</footer>
`;
